mage/mage/readability_check.clj

```diff
@@ -167,8 +167,7 @@
   (check "bin/build/test/i18n/create_artifacts/backend_test.clj" 29)
   ;; => {:readable true, :starting-at "\n(deftest ^:parallel backend-message?"}
 
-
-  ;; this should be a test:
+;; this should be a test:
```

Comment:
Again, this is a whitespace change from the new pre-commit hook. I didn't change this file.

Reply:
👋 Whoops, that's on me!